docs/csharp/programming-guide/interfaces/index.md

```diff
@@ -7,86 +7,78 @@ helpviewer_keywords:
+ [How to: Explicitly Implement Interface Members](how-to-explicitly-implement-interface-members.md)  
  Provides an example of how to explicitly implement members of interfaces.  
   
- [How to: Explicitly Implement Members of Two Interfaces](../../../csharp/programming-guide/interfaces/how-to-explicitly-implement-members-of-two-interfaces.md)  
+ [How to: Explicitly Implement Members of Two Interfaces](how-to-explicitly-implement-members-of-two-interfaces.md)  
  Provides an example of how to explicitly implement members of interfaces with inheritance.  
   
-##  <a name="BKMK_RelatedSections"></a> Related Sections  
-  
--   [Interface Properties](../../../csharp/programming-guide/classes-and-structs/interface-properties.md)  
-  
--   [Indexers in Interfaces](../../../csharp/programming-guide/indexers/indexers-in-interfaces.md)  
-  
--   [How to:  Implement Interface Events](../../../csharp/programming-guide/events/how-to-implement-interface-events.md)  
-  
--   [Classes and Structs](../../../csharp/programming-guide/classes-and-structs/index.md)  
-  
--   [Inheritance](../../../csharp/programming-guide/classes-and-structs/inheritance.md)  
-  
--   [Methods](../../../csharp/programming-guide/classes-and-structs/methods.md)  
-  
--   [Polymorphism](../../../csharp/programming-guide/classes-and-structs/polymorphism.md)  
-  
--   [Abstract and Sealed Classes and Class Members](../../../csharp/programming-guide/classes-and-structs/abstract-and-sealed-classes-and-class-members.md)  
-  
--   [Properties](../../../csharp/programming-guide/classes-and-structs/properties.md)  
-  
--   [Events](../../../csharp/programming-guide/events/index.md)  
-  
--   [Indexers](../../../csharp/programming-guide/indexers/index.md)  
-  
-## Featured Book Chapter  
- [Interfaces](http://msdn.microsoft.com/library/orm-9780596521066-01-13.aspx) in [Learning C# 3.0: Master the Fundamentals of C# 3.0](http://msdn.microsoft.com/library/orm-9780596521066-01.aspx)  
-  
-## See Also  
- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
- [Inheritance](../../../csharp/programming-guide/classes-and-structs/inheritance.md)
+##  <a name="BKMK_RelatedSections"></a> Related Sections
+
+- [Interface Properties](../classes-and-structs/interface-properties.md)  
+- [Indexers in Interfaces](../indexers/indexers-in-interfaces.md)  
+- [How to:  Implement Interface Events](../events/how-to-implement-interface-events.md)  
+- [Classes and Structs](../classes-and-structs/index.md)  
+- [Inheritance](../classes-and-structs/inheritance.md)  
+- [Methods](../classes-and-structs/methods.md)  
+- [Polymorphism](../classes-and-structs/polymorphism.md)  
+- [Abstract and Sealed Classes and Class Members](../classes-and-structs/abstract-and-sealed-classes-and-class-members.md)  
+- [Properties](../classes-and-structs/properties.md)  
+- [Events](../events/index.md)  
+- [Indexers](../indexers/index.md)  
+  
+## Featured Book Chapter
+
```

Comment:
not related to your changes, but do you want to update these msdn links to docs?